.github/workflows/test-ghcup.yaml

```diff
@@ -163,3 +165,21 @@ jobs:
         run: |
           cat $(stack path --stack-root)/hooks/ghc-install.sh
         shell: bash
+
+  make:
+    runs-on: ubuntu-latest
+    container:
+      image: rockylinux:9
+    steps:
+      - run: |
+          dnf install -y git nodejs make
+      - uses: actions/checkout@v4
+      - run: |
+          set -eux
+          pushd ghcup
+          npx bun install
+          BUN="npx bun" make
+          popd
+          git config --global --add safe.directory $PWD
+          git status
+          git diff --exit-code
```

Comment:
For some reason I get a different `index.js` here although the source files have not changed.

Locally, I can't reproduce, so something from the environment must be leaking into the output.